pmd-dist/src/main/scripts/run.sh

```diff
@@ -141,5 +141,15 @@ cygwin_paths
 
 java_heapsize_settings
 
-java ${HEAPSIZE} $(jre_specific_vm_options) -cp "${classpath}" "${CLASSNAME}" "$@"
+# Create a temporary file to direct the java command's output to
+temp_file=$(mktemp)
+java ${HEAPSIZE} $(jre_specific_vm_options) -cp "${classpath}" "${CLASSNAME}" "$@" 2>&1 | tee -a temp_file
+exit_code=${PIPESTATUS[0]}
+
+# Did the java command fail because we're missing net.sourceforge.pmd.util.fxdesigner.Designer?
+if grep -q "net.sourceforge.pmd.util.fxdesigner.Designer" temp_file; then
+  echo "It looks like you're missing the class net.sourceforge.pmd.util.fxdesigner.Designer. A solution might be available here: https://github.com/pmd/pmd/issues/962"
```

Comment:
I'd rather say something along the lines of "You seem to be missing JavaFX runtime" rather than that they are missing a PMD class... I think it's both more accurate and more likely to send people in the right direction.

Reply:
That makes sense. Change made in https://github.com/pmd/pmd/pull/1146/commits/bb8b46a694be82a9f55a3e71bdc0e036727a85a0. Please feel free to nitpick on the exact message.